hybrasyl/Game.cs

```diff
@@ -201,27 +201,35 @@ public static void Main(string[] args)
             LogLevel = Hybrasyl.Constants.DEFAULT_LOG_LEVEL;
             Assemblyinfo = new AssemblyInfo(Assembly.GetEntryAssembly());
 
-            
             Constants.DataDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "Hybrasyl");
+            
 
-            if (!Directory.Exists(Constants.DataDirectory))
+            if (Directory.Exists(Constants.DataDirectory))

```

Comment:
If you're creating all the other directories, might as well create the main directory too.

Reply:
Main directory gets created by the Logger.